Atomic/syscontainers.py

```diff
@@ -1646,7 +1646,13 @@ def _inspect_system_branch(self, repo, imagebranch):
             virtual_size = self._get_virtual_size(repo, manifest)
             if 'Labels' in manifest:
                 labels = manifest['Labels']
-            else:
+            elif 'history' in manifest:
```

Comment:
Might be time to educate Tom, but should this be an if rather than an elif?

Reply:
no, I'd like to take this branch only if the first 'if' was not satisfied.

If the labels are found in `manifest['Labels']` we don't need to look into 'history'.

Reply:
@giuseppe Thanks for the info.  FYI some of the tests aren't looking happy.

Reply:
it is because of storage :( it is failing since long time but it should not be related to this PR